libs/sdk-backend-tiger/src/backend/organization/settings.ts

```diff
@@ -18,15 +18,24 @@ export class OrganizationSettingsService
         return this.setSetting("WHITE_LABELING", whiteLabeling);
     }
 
+    public async setTimezone(timezone: string): Promise<void> {
+        return this.setSetting("TIMEZONE", { value: timezone });
+    }
+
+    public async setDateFormat(dateFormat: string): Promise<void> {
+        return this.setSetting("FORMAT_LOCALE", { value: dateFormat });
+    }
+
     public async getSettings(): Promise<ISettings> {
         const { data } = await this.authCall(async (client) =>
             client.entities.getAllEntitiesOrganizationSettings({}),
         );
 
         return data.data.reduce((result: ISettings, setting) => {
+            const id = setting.attributes?.type === "WHITE_LABELING" ? "whiteLabeling" : setting.id;
```

Comment:
Sry that i enter into discussion so alte, but this is not good approach. Every setting has dynamic id now on tiger, not only white labeling, so basically we need to this mapping based on type, but not in SDK. This must be done on client side or introduce new api that return settings mapped by type, this was a public api change, that is wrong.